strawberryfields/apps/qchem/utils.py

```diff
@@ -96,3 +96,59 @@ def duschinsky(
     delta = np.array(d @ l0_inv)
 
     return U, delta
+
+
+def read_gamess(file) -> Tuple[np.ndarray, np.ndarray, np.ndarray, np.ndarray]:
+    r"""Reads molecular data from the output file generated by the GAMESS quantum chemistry package.
+
+    This function extracts the atomic coordinates, atomic masses, vibrational frequencies,
+    and normal modes of a molecule from the output file of a vibrational frequency calculation
+    performed with the GAMESS quantum chemistry package.
+
+    **Example usage:**
+
+    >>> r, m, w, l = read_gamess('../BH_frq.out')
+    >>> r
+    array([0.000000, 0.000000, 0.000000, 1.253604, 0.000000, 0.000000]
+
+    Args:
+        file: name and the path of the GAMESS output file
+
+    Returns:
+        tuple[array, array, array, array]: atomic coordinates, atomic masses, normal mode
+        frequencies, normal modes
+    """
+    with open(file, "r") as f:
+
+        r = []
+        m = []
+        w = []
+        l = []
+
+        for line in f:
+
+            if "INPUT CARD> $data" in line or "INPUT CARD> $DATA" in line:
```

Comment:
So GAMESS sometimes uses uppercase and other times it does not? :thinking:

Reply:
This part is actually defined by the user. Some users prefer lowercase and some use uppercase.

Reply:
The keywords used in the input files of GAMESS are not case sensitive.